.hound.ruby.yml

```diff
@@ -1,13 +1,15 @@
 inherit_from: .hound.suse.yml
 
 AllCops:
-  RunRailsCops: true
+  TargetRubyVersion: 2.1
+  Rails:
+    Enabled: true
```

Comment:
I agree we don't really need it, but given the fact that it was activated before (`RunRailsCops: true`) and it din't create issues I would just let keep it here to make it easier to sync the configs (just copy and commit).